app/controllers/forms_controller.rb

```diff
@@ -22,6 +23,12 @@ def show
   end
 
   def submit
+    if @current_user.present?
+      @current_user.update_attributes(
+        params.require(:user)
+        .permit(:first_name, :last_name, :email)
+      )
+    end
```

Comment:
Invalid for the code: No. Invalid in terms of someone putting in a fake name? Sure?